views/default/page/elements/comments.php

```diff
@@ -19,22 +19,25 @@
 	$limit = elgg_trigger_plugin_hook('config', 'comments_per_page', [], 25);
 }
 
-$id = '';
-if (isset($vars['id'])) {
-	$id = "id=\"{$vars['id']}\"";
-}
+$attr = [
+	'id' => elgg_extract('id', $vars, 'elgg-comments')
+];
 
 $class = 'elgg-comments';
 if (isset($vars['class'])) {
 	$class = "$class {$vars['class']}";
 }
+$attr['class'] = $class;
```

Comment:
@jdalsem has an elegant refactoring. Replace this and the previous 4 lines with:
```php
$attr['class'] = (array) elgg_extract('class', $vars, []);
$attr['class'][] = 'elgg-comments';
```